From 2f0d273b6353e4896adc86d766edcac5d10386fe Mon Sep 17 00:00:00 2001
From: Reuben Kuhnert <reuben.kuhnert@cloudera.com>
Date: Tue, 8 Sep 2015 11:37:20 -0500
Subject: [PATCH 0257/1164] CLOUDERA-BUILD: OPSAPS-28063:
 hive.stats.collect.scancols needs to be set to
 true in HS2 to allow column level priviliges

---
 .../java/org/apache/hadoop/hive/conf/HiveConf.java |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/src/java/org/apache/hadoop/hive/conf/HiveConf.java b/common/src/java/org/apache/hadoop/hive/conf/HiveConf.java
index 7417f24..d4ce9e9 100644
--- a/common/src/java/org/apache/hadoop/hive/conf/HiveConf.java
+++ b/common/src/java/org/apache/hadoop/hive/conf/HiveConf.java
@@ -1243,7 +1243,7 @@ public void setSparkConfigUpdated(boolean isSparkConfigUpdated) {
         "Whether join and group by keys on tables are derived and maintained in the QueryPlan.\n" +
         "This is useful to identify how tables are accessed and to determine if they should be bucketed."),
     // Collect column access information
-    HIVE_STATS_COLLECT_SCANCOLS("hive.stats.collect.scancols", false,
+    HIVE_STATS_COLLECT_SCANCOLS("hive.stats.collect.scancols", true,
         "Whether column accesses are tracked in the QueryPlan.\n" +
         "This is useful to identify how tables are accessed and to determine if there are wasted columns that can be trimmed."),
     // standard error allowed for ndv estimates. A lower value indicates higher accuracy and a
-- 
1.7.9.5

